packages/website/src/pages/related-tools.tsx

```diff
@@ -2,96 +2,183 @@ import CarbonAd from 'src/components/CarbonAd'
-    description: 'Gitmoji tool for git commit messages in VS Code',
-    link: 'https://github.com/seatonjiang/gitmoji-vscode',
-  },
-  {
-    name: 'gitmoji-intellij-plugin',
-    description:
-      'A Jetbrains suite plugin to easily add gitmoji when committing',
-    link: 'https://plugins.jetbrains.com/plugin/12383-gitmoji-plus-commit-button',
-  },
-  {
-    name: 'gitmoji-sublimetext',
-    description: 'A Sublime Text plugin to add emojis in git commit messages.',
-    link: 'https://packagecontrol.io/packages/Gitmoji',
-  },
-  {
-    name: 'gitimoji',
-    description: 'A Gitmoji App for macOS.',
-    link: 'https://github.com/TimoZacherl/gitimoji',
-  },
-  {
-    name: 'gitmoji-atom',
-    description: 'Gitmoji for Atom',
-    link: 'https://github.com/ThatXliner/gitmoji-atom',
-  },
-  {
-    name: 'gitmoji-regex',
-    description: 'A Gitmoji::Regex for Ruby.',
-    link: 'https://github.com/pboling/gitmoji-regex',
-  },
-  {
-    name: 'traymoji',
-    description: 'A Electron Tray App for Gitmojis',
-    link: 'https://github.com/CoenWarmer/traymoji',
-  },
-  {
-    name: 'alfred-gitmoji',
-    description: 'Gitmoji Workflow for Alfred',
-    link: 'https://github.com/techouse/alfred-gitmoji',
-  },
-  {
-    name: 'gitmojiapp',
-    description: 'A Flutter Gitmoji App for macOS, Linux, and Windows',
-    link: 'https://github.com/patrick-fu/GitmojiApp',
-  },
-  {
-    name: 'githubmoji',
-    description:
-      'A Firefox addon that adds a predictive gitmoji picker to GitHub commit message inputs.',
-    link: 'https://github.com/ma1ted/githubmoji',
-  },
-  {
-    name: 'gitmoji-changelog-action',
-    description: 'GitHub Action for gitmoji-changelog',
-    link: 'https://github.com/sercanuste/gitmoji-changelog-action',
-  },
-  {
-    name: 'raycast-gitmoji-search',
-    description: 'Gitmoji extension for Raycast',
-    link: 'https://www.raycast.com/ricoberger/gitmoji',
-  },
-  {
-    name: 'gitmoji-fuzzy-hook',
-    description:
-      'Fuzzy finder git-commit-hook for prepending a gitmoji (cmd & GUI).',
-    link: 'https://gitlab.com/raabf/gitmoji-fuzzy-hook',
-  },
-  {
-    name: 'genmoji',
-    description: 'Gitmoji commit message generation using AI',
-    link: 'https://github.com/segersniels/genmoji',
-  },
+	{
+		emoji: '📦',
+		title: 'Apps',
+		items: [
+			{
+				name: 'gitimoji',
+				description: 'A Gitmoji App for macOS',
+				link: 'https://github.com/TimoZacherl/gitimoji',
+			},
+			{
+				name: 'gitmojiapp',
+				description: 'A Flutter Gitmoji App for macOS, Linux, and Windows',
+				link: 'https://github.com/patrick-fu/GitmojiApp',
+			},
+			{
+				name: 'gitmojiBar',
+				description: 'MacOS menubar for easy copy gitmoji',
+				link: 'https://github.com/techinpark/gitmojiBar',
+			},
+			{
+				name: 'traymoji',
+				description: 'A Electron Tray App for Gitmojis',
+				link: 'https://github.com/CoenWarmer/traymoji',
+			},
+		],
+	},
+	{
+		emoji: '🏭',
+		title: 'Automatic generators',
```

Comment:
Maybe `DevTools` is a better name?

```suggestion
		title: 'DevTools',
```